### PR TITLE
Add spec and api stub

### PR DESCRIPTION
**Pull request by john-g-g**
Closes #SD-348

### CODE REVIEW

**Comment by bnchdrff** [comment]
@john-g-g looks like tests are failing because you added a package but did not commit the changes to `yarn.lock` - you can fix this by adding the package with yarn (again maybe) and then committing the changes it will make to the lockfile

File: src/user-search.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {JsonRpcAuthMethodContext as JCtx, JsonRpcError} from '@steemit/koa-jsonrpc'
+import * as config from 'config'
+import {ValidationError} from 'sequelize'
```
Comment (gl2748) [inline src/user-search.ts]: Is ValidationError used here?

**Comment by gl2748** [comment]
@john-g-g please resolve conflicts, then this looks GTG.

**Comment by gl2748** [comment]
Actually NVM, I resolved it.

**Pull request merged**